src/rgw/rgw_appmain.cc

```diff
@@ -449,7 +449,6 @@ int rgw::AppMain::init_frontends2(RGWLib* rgwlib)
 
   // initialize RGWProcessEnv
   env.rest = &rest;
-  env.olog = olog;
```

Comment:
OK, this was initially confusing but we carried this assignment to line 410 looks like it and `olog` pointer itself is not used but only `env.olog`?